src/components/ActivityCard.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-container pa-0>
+    <v-card>
+      <v-img
+        :height="imageHeight"
+        :src="imgSrc"
+      ></v-img>
+      <v-card-title class="no-padding-bottom">
+        <div class="text-truncate">
+          <p class="title text-truncate">
+            {{ title }}
+          </p>
+          <p class="body-2 grey--text text-truncate">
+            {{ description }}
+          </p>
+        </div>
+      </v-card-title>
+      <v-card-actions class="no-padding-top">
+        <v-btn flat>
+          Saber mais
+        </v-btn>
+        <v-spacer></v-spacer>
+        <v-btn :color="buttonColor">
+          {{ finalized ? 'Finalizada' : 'Feito' }}
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-container>
+</template>
+
+<script>
+
+export default {
+  name: 'activity-card',
+  props: {
+    description: {
+      type: String,
+      required: true,
```

Comment:
In the future, rendering this component will depends on a Promise. We can keep this but the parent component should have and "v-if" condition. Anyway, it is just a comment